frontend_build/src/parse_bundle_plugin.js

```diff
@@ -106,27 +121,13 @@ var parseBundlePlugin = function(data, base_dir) {
       __events: JSON.stringify(data.events || {}),
       __once: JSON.stringify(data.once || {}),
       __version: JSON.stringify(data.version),
+      __publicPath: JSON.stringify(publicPath),
```

Comment:
Allow the publicPath that a script will be served on to be referenced in built JS.